### ISSUE TITLE
Remove Heartbeat prompting from Test Pilot add-on.

### ISSUE THREAD

**Issue by chuckharmston** [mozilla/testpilot#2561]
Due to two factors:

- Photon drastically changing the browser chrome CSS.
- The desire to remove functionality from our add-on.

We're going to remove the Heartbeat notification prompts.

**Comment by chuckharmston** [comment]
<img width="1543" alt="screen shot 2017-07-01 at 18 11 31" src="https://user-images.githubusercontent.com/23885/27766563-ea93b538-5e88-11e7-95a9-e4b8c0af0ae5.png">

This is now happening on Dev Edition as well as Nightly.

**Comment by lmorchard** [comment]
This invalidates #2512

Alternatively - and this might be a terrible idea, because it purposefully goes against the non-spammy grain of webextension APIs - in working toward #2423, I figured out how to [set a popup and self-trigger the browser action button](https://github.com/lmorchard/testpilot/commit/d966a24b10625d79c3cbd21856f3b801ed059b8f#diff-270f74550696a83b1cb4e9a03141b730R14) from an embedded webextension. The button click [happens in the privileged bootstrapped part](https://github.com/lmorchard/testpilot/commit/d966a24b10625d79c3cbd21856f3b801ed059b8f#diff-014485cea24125f21f546041b718343dR78) in response to [a message from the webextension background script](https://github.com/lmorchard/testpilot/commit/d966a24b10625d79c3cbd21856f3b801ed059b8f#diff-270f74550696a83b1cb4e9a03141b730R17). The stubbed out survey page [clears the popup](https://github.com/lmorchard/testpilot/commit/d966a24b10625d79c3cbd21856f3b801ed059b8f#diff-c1f1e254f07e475acc32bcd6a8e2bae1R1) on display, so that the next browser action click just opens the Test Pilot site. 

TL;DR: Convoluted behind the scenes, but could let us redesign the survey prompt as a webextension popup. /cc @johngruen 

(FWIW, we could keep this in our back pocket for launching the rewritten add-on and revisit it later. Or, even forget I ever thought of it.)

**Comment by ghost** [comment]
It sounds like this code just needs to get removed.

**Comment by lmorchard** [comment]
FWIW, the addon i'm working on for #2423 doesn't include any feedback prompting yet. So we could consider that a fix for this

**Comment by fzzzy** [comment]
I think this was fixed by the remove-sdk addon.
